tree/treeplayer/inc/TTreeReaderArray.h

```diff
@@ -33,7 +33,7 @@ namespace Internal {
    public:
       TTreeReaderArrayBase(TTreeReader* reader, const char* branchname,
                            TDictionary* dict):
-         TTreeReaderValueBase(reader, branchname, dict), fImpl(0) {}
```

Comment:
`fImpl(0)` is replaced by the `unique_ptr` mechanics?

Reply:
Indeed, nice!